src/components/Input.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import Button from './Button'
+import { nanoid } from 'nanoid'
+
+function Input({ onCreateCard }) {
+  function handleSubmit(event) {
+    event.preventDefault()
+    const form = event.target
+    const { text } = form.elements
+    console.log(text.value)
+    /* setCardData({ ...cardData, text: text.value }) */
```

Comment:
the console.log and the comment are good for testing, but they shouldn't be in the final code